src/esperanto/NewsletterBundle/Twig/NewsletterRender.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace esperanto\NewsletterBundle\Twig;
+
+use Symfony\Component\DependencyInjection\Container;
+use Symfony\Component\Templating\EngineInterface;
+
+class NewsletterRender extends \Twig_Extension
+{
+    /**
+     * @var Container
+     */
+    protected $container;
+
+    /**
+     * @var EngineInterface
+     */
+    protected $templateEngine;
+
+    /**
+     * @param Container $container
+     * @param $template string
+     */
+    public function __construct(Container $container)
+    {
+        $this->container = $container;
+    }
+
+    public function getFunctions()
+    {
+        return array(
+            new \Twig_SimpleFunction('newsletter_render', array($this, 'render'), array('is_safe' => array('html'))),
+        );
+    }
+
+    public function render($template)
```

Comment:
$template should be optional. If no one is defined we use a standard template, which should be define it the NewsletterBundle